src/api/services/emissionsService.ts

```diff
@@ -12,7 +12,7 @@ import { DefaultEmissions } from '../types'
 import { prisma } from '../../lib/prisma'
 import { emissionsArgs } from '../args'
 
-const TONNES_CO2_UNIT = 'tCO2e'
+export const TONNES_CO2E_UNIT = 'tCO2e'
```

Comment:
Maybe this should be the default unit used for the `emissionUnitSchema`? If we move this constant to the same module as the `emissionUnitSchema`, we could also keep related values together.

Or should this be a part of the config? I'm happy with keeping it in the common schemas to keep it simple.